core/strings/conversion.odin

```diff
@@ -57,14 +69,19 @@ to_valid_utf8 :: proc(s, replacement: string, allocator := context.allocator) ->
 	}
 	return to_string(b)
 }
-
-/*
-	returns the input string `s` with all runes set to lowered case
-	always allocates using the `allocator`
-
-	strings.to_lower("test") -> test	
-	strings.to_lower("Test") -> test	
-*/
+// Converts the input string `s` to all lowercase characters.
+//
+// *Allocates Using Provided Allocator*
+//
+// Inputs:
+// - s: Input string to be converted.
+// - allocator: (default: context.allocator).
+//
+// Usage:
```

Comment:
You should use Example: rather than Usage:
This will generate a colourised code block on the website that is collapsible.